docs/decisions/0010-include-eslint-plugin-vuejs-accessibility.md

```diff
@@ -0,0 +1,88 @@
+
+## Decision Drivers <!-- optional -->
+
+- We want to ensure that our Vue.js applications are accessible to all users.
+<!-- numbers of drivers can vary -->
+
+## Considered Options
+
+- Do not include an eslint-plugin for accessibility
+- [eslint-plugin-vuejs-accessibility](https://www.npmjs.com/package/eslint-plugin-vuejs-accessibility) 
+- [eslint-plugin-vue-a11y](https://www.npmjs.com/package/eslint-plugin-vue-a11y) 
+<!-- numbers of options can vary -->
+
+## Decision Outcome
+
+TBD
+
+<!-- Chosen option: "[option 1]", because [justification. e.g., only option, which meets k.o. criterion decision driver | which resolves force force | … | comes out best (see below)]. -->
+
+### Positive Consequences <!-- optional -->
+
+- Catching the accessibility issues in our Vue.js applications before they are deployed to production.
+- Helping developers to write more accessible Vue.js applications.
+- Reducing future remediation costs.
+
+### Negative Consequences <!-- optional -->
+
+- Some devs may not know how to fix accessibility issues in Vue.js applications, and this may cause some friction in development or result in a number of rules being disabled.
+
+## Pros and Cons of the Options <!-- optional -->
+
+### Do not include an eslint-plugin for accessibility
+
+- Good, because no changes 
+- Bad, because we will miss out on catching accessibility issues in our Vue.js applications before they are deployed to production.
+<!-- numbers of pros and cons can vary -->
+
+### eslint-plugin-vuejs-accessibility
+
+- [NPM package - eslint-plugin-vuejs-accessibility](https://www.npmjs.com/package/eslint-plugin-vuejs-accessibility) 
+  - [List of rules](https://vue-a11y.github.io/eslint-plugin-vuejs-accessibility/rule-overview/)
+  - [Overview of the Vue A11y project](https://vue-a11y.com/project/#introduction)
+  - [Releases](https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/releases)
+
+- Good, because it is being actively maintained and has 231K weekly downloads.
+- Good, because we would catch accessibility issues in our Vue.js applications before they are deployed to production.
+<!-- numbers of pros and cons can vary -->
+
+### eslint-plugin-vue-a11y
+
+[NPM package - eslint-plugin-vue-a11y](https://www.npmjs.com/package/eslint-plugin-vue-a11y)  <!-- optional -->
+
+- Bad, because it is not being actively maintained - was last published 5 years ago and only has 3k weekly downloads.
+- Good, because we would catch accessibility issues in our Vue.js applications before they are deployed to production.
+<!-- numbers of pros and cons can vary -->
+
+## Code Examples
+
+### Option 1 Code Example <!-- optional -->
+
+n/a - no changes from current setup
+
+### Option 2 Code Example <!-- optional -->
+
+[Changes are in this PR with the Decision Record - #3634](https://github.com/usdigitalresponse/usdr-gost/pull/3634/files)
+
+I can separate out the changes into separate PR if needed.
```

Comment:
Assuming we end up adopting at least one of the recommended linters, I don't think it should be necessary to separate the code changes from this PR into a separate PR. However, I do think that we should make a plan to address the newly-introduced `eslint-disable` directives, which might entail having individual issues (either per-file, or else something more aligned with perceived level of effort) along with a parent issue we can use to track burn-down progress.

Reply:
Sounds good.  If this is adopted, I plan to work on addressing the issues.   Before merging, I'll create tech debt tickets and add them to the disable lines.  I agree with your suggestion of one ticket per page.

One of the findings - autofocus would need to be reviewed w/ UX as it would change the behavior of the application.  I don't know if this behavior was added by devs or specified by UX.

* https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus

> [Accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus#accessibility_concerns)
> Automatically focusing a form control can confuse visually-impaired people using screen-reading technology and people with cognitive impairments. When autofocus is assigned, screen-readers "teleport" their user to the form control without warning them beforehand.
> 
> Use careful consideration for accessibility when applying the autofocus attribute. Automatically focusing on a control can cause the page to scroll on load. The focus can also cause dynamic keyboards to display on some touch devices. While a screen reader will announce the label of the form control receiving focus, the screen reader will not announce anything before the label, and the sighted user on a small device will equally miss the context created by the preceding content.